### PR TITLE
feat: NVIDIA Jetson integration documentation

### PR DESCRIPTION
**Pull request by sjyothi54**
**Ticket:** https://new-relic.atlassian.net/browse/NR-278693
**Description:** Created NVIDIA Jetson integration documentation

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by akristen** [comment]
Thanks so much for adding this new doc @sjyothi54 ! I'll review it this week and get it merged and released within 1-3 days!!

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+tags:
+    - New Relic integrations
+    - NVIDIA Jetson integration
+metaDescription: Use New Relic infrastructure agent to get a dashboard with metrics from your NVIDIA Jetson.
+freshnessValidatedDate: 2024-06-18
+---
+import jetsonDashboard from 'images/infrastructure_screenshot_full-nvidia-jetson-dashboard.webp'
+
+
+Our NVIDIA Jetson integration collects and sends data about your tegrastats utility to New Relic, where you can monitor CPU, GPU, RAM, and temperature readings about your NVIDIA Jetson device.
```
Comment (akristen) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx:12]: @sjyothi54 I looked up NVIDIA Jetson and saw that it's both a hardware device and may also include some software. It seems like this integration is mostly concerned with the hardware side of the device... Is that accurate?

Comment (sjyothi54) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx:12]: @akristen Yes, Jetson is a hardware device. Users of Jetson can obtain metrics, such as tegrastats, by default. By using our nri-flex integration, they can seamlessly send these metrics to New Relic.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    src={jetsonDashboard}
+/>
+<figcaption>
+After setting up our NVIDIA Jetson integration, we give you a dashboard for your NVIDIA Jetson metrics. 
+</figcaption>
+
+
+<Steps>
+    <Step>
+## Install the infrastructure agent [#infra-install]
```
Comment (akristen) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx:26]:
```suggestion
### Install the infrastructure agent [#infra-install]
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+To use the NVIDIA Jetson integration, you need to first [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) onto your host. The infrastructure agent monitors the host itself, while the integration itself extends your monitoring to include NVIDIA Jetson-specific data.
+    </Step>
+    <Step>
+## Save tegrastats to a file [#save-tegrastats]
```
Comment (akristen) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx:31]:
```suggestion
### Save tegrastats to a file [#save-tegrastats]
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+From your shell environment, change to the tegrastats directory and run this command to save NVIDIA Jetson metrics at 5 second intervals:
+
+```shell
+sudo tegrastats --interval 5000 --logfile tegrastats
+```    
+    </Step>
+    <Step>
+
+## Enable the NVIDIA Jetson integration with nri-flex [#enable-jetson]
```
Comment (akristen) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx:41]:
```suggestion
### Enable the NVIDIA Jetson integration with nri-flex [#enable-jetson]
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+                  split: horizontal
+                  set_header: [pom_in_currect, pom_in_peak, pom_gpu_used, pom_gpu_total, pom_cpu_used, pom_cpu_total]
+                  regex_match: true
+                  split_by: "POM_5V_IN (\\d+)/(\\d+) POM_5V_GPU (\\d+)/(\\d+) POM_5V_CPU (\\d+)/(\\d+)"            
+    ```
+    </Step>
+    <Step>
+
+
+## Restart the New Relic infrastructure agent [#restart-agent]
```
Comment (akristen) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx:126]:
```suggestion
### Restart the New Relic infrastructure agent [#restart-agent]
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+
+    ```shell
+    sudo systemctl restart newrelic-infra.service
+    ```
+
+Wait a couple of minutes as your application sends metrics to [one.newrelic.com](https://one.newrelic.com).        
+    </Step>
+    <Step>
+
+## Find your data [#find-data]
```
Comment (akristen) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx:138]:
```suggestion
### Find your data [#find-data]
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+With NVIDIA Jetson reporting data, you can also build queries to learn more about your device's performance:
+
+1. From any page in **[one.newrelic.com](https://one.newrelic.com)**, click **Query your data** at the bottom of your browser. 
+1. Build your query. For example, this is an example NRQL query to view the GPU temperature:
+
+  ```sql
+  SELECT latest(gpu_temp) as 'GPU temperature (C)' from jetsonTegrastats 
+  ```    
+
+## What's next?
```
Comment (akristen) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx:161]:
```suggestion
## What's next? [#whats-next]
```

**Review by akristen — APPROVED** [review]

**Pull request merged**